docs/apache-airflow/concepts/scheduler.rst

```diff
@@ -138,12 +141,101 @@ The following databases are fully supported and provide an "optimal" experience:
 
   Microsoft SQLServer has not been tested with HA.
 
+
+Fine-tuning your Scheduler
+--------------------------
+
+When you deploy Airflow in production you often would like to optimize its performance and
+fine-tune Scheduler behaviour. Firs of all you need to remember that Scheduler performs two
+operations:
+
+* continuously parses DAG files and updates their starts in ``Serialized DAG`` form in the database
+* continuously finds and schedules for execution the next tasks to run and sends those tasks for
+  execution to the executor you have configured
```

Comment:
Nothing wrong here, but would try to avoid internal details not necessary to the story

```suggestion
1. continuously parsing DAG files and synchronizing with the DAG in the database
2. continuously scheduling tasks for execution
```